src/test/java/core/basesyntax/CalculatorTest.java

```diff
@@ -0,0 +1,375 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class CalculatorTest {
+    private static final double DELTA = 0.0001;
+    private static CalculatorService calculator;
+
+    @BeforeAll
+    static void beforeAll() {
+        calculator = new Calculator();
+    }
+
+    @Test
+    void additionTwoPositiveOperands_Ok() {
```

Comment:
please add the name of the method that you are testing, to all test methods
```suggestion
    void calculate_additionTwoPositiveOperands_Ok() {
```

Reply:
Thanks, ✔️